charts/nublado2/values.yaml

```diff
@@ -216,11 +216,14 @@ config:
         TAP_ROUTE: /api/tap
         SODA_ROUTE: /api/image/soda
         WORKFLOW_ROUTE: /wf
-        AUTO_REPO_URLS: "https://github.com/lsst-sqre/notebook-demo"
+        AUTO_REPO_URLS: https://github.com/lsst-sqre/notebook-demo
+        PGPASSFILE: /opt/lsst/software/jupyterlab/butler-secret/postgres-credentials.txt
```

Comment:
Do we still want to use a file despite the warning, or go back to putting the secret in an environment variable to avoid the warning?

Reply:
Let's start here.  Because of the way we're injecting the environment I think it's going to take further work in nublado2 to mix-and-match the environment-as-configmap and environment-as-secret.  This will at least get us off the ground.